addons/web/static/src/core/debug/debug_menu_items.xml

```diff
@@ -112,4 +112,13 @@
             </table>
         </Dialog>
     </t>
+
+    <t t-name="web.DebugMenu.GetViewDialog">
+        <Dialog title="title">
+            <t t-set-slot="footer">
```

Comment:
Can we put the footer slot after the "pre" tag? Makes more sense to have the footer at the end.